code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm

```diff
@@ -139,6 +148,12 @@
 	. += span_notice(" -Use a multitool with left-click to change the piping layer and right-click to change the piping color.")
 	. += span_notice(" -[EXAMINE_HINT("AltClick")] to cycle between temperaure ranges.")
 	. += span_notice(" -[EXAMINE_HINT("CtrlClick")] to toggle on/off.")
+	if(pyro_anomaly_core)
+		. += span_notice(" -There is a pyroclastic anomaly core slotted into the back of the machine, it pumps massive amounts of heat into the machine.")
+	else
+		. += span_notice(" -There seems to be a slot for something in the back, what could it be?")
+	if(istype(src, /obj/machinery/atmospherics/components/unary/thermomachine/anomalous))
+		. += span_notice(" -This machine has been configured to only produce heat and cannot cool gas down")
```

Comment:
This should be on the anomalous' examine text instead

Reply:
Fixed, I do not know the normal naming conventions so as of right now it is appended to the end of the examine.